it/builder/builder-image/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.0)
```

Comment:
How are we using these C related files in these integration test? Sorry, since there is no readme It's a bit difficult at a quick look.

Reply:
Doesn't [the comment](https://github.com/fabric8io/docker-maven-plugin/blob/8e2cc922908fb7fc1cc5484a0e5f10aef3e8944f/it/builder/pom.xml#L6) (description of this integration test) in [it/builder/pom.xml](https://github.com/fabric8io/docker-maven-plugin/blob/8e2cc922908fb7fc1cc5484a0e5f10aef3e8944f/it/builder/pom.xml) answer your question? Do you feel that we need to put (duplicate?) description of the integration test in another place (I followed existing integration tests which description is placed at the top POM of the test) or / and extend it?

Reply:
No, I think it's okay. I just didn't notice that you had added a header there.